.TAOS-CI/config/config-plugins-prebuild.sh

```diff
@@ -11,6 +11,7 @@
 
 ##### Set environment for plugin modules of prebuild group
 declare -i idx=-1
+declare clang_option="cxx-only"
```

Comment:
IMO, it would be better for `config/config-environment.sh` to handle this option, not here.

Reply:
Your comment is applied. Thanks.